locale/en/docs/guides/domain-postmortem.md

```diff
@@ -57,7 +57,7 @@ the active domain to hijack the error:
 const domain = require('domain');
 const net = require('net');
 const d = domain.create();
-d.on('error', (err) => console.error(err.message));
+d.on('error', (err) => { console.error(err.message); });
```

Comment:
It seems to me that omission of curly braces infers implicit return. Let me know if I should revert.